android/library/src/main/java/io/logto/client/LogtoClient.kt

```diff
@@ -43,31 +44,60 @@ open class LogtoClient(
         return urlBuilder.buildString()
     }
 
-    suspend fun fetchOidcConfiguration(): OidcConfiguration =
-        logtoService.fetchOidcConfiguration(logtoConfig.domain)
-
     suspend fun grantTokenByAuthorizationCode(
         tokenEndpoint: String,
         authorizationCode: String,
         codeVerifier: String,
-    ): TokenSet = logtoService.grantTokenByAuthorizationCode(
-        tokenEndpoint = tokenEndpoint,
-        clientId = logtoConfig.clientId,
-        redirectUri = logtoConfig.redirectUri,
-        code = authorizationCode,
-        codeVerifier = codeVerifier
-    )
+    ): TokenSet {
+        val tokenSetParameters = logtoService.grantTokenByAuthorizationCode(
+            tokenEndpoint = tokenEndpoint,
+            clientId = logtoConfig.clientId,
+            redirectUri = logtoConfig.redirectUri,
+            code = authorizationCode,
+            codeVerifier = codeVerifier
+        )
+
+        val jsonWebKeySet = getJsonWebKeySet()
+        tokenSetParameters.verifyIdToken(logtoConfig.clientId, jsonWebKeySet)
```

Comment:
could call `TokenUtils.verifyIdToken` directly

Reply:
it's hard to mock `TokenUtils` in ut.
Will fix this if we change our ut tools some day